server/routes/home.ts

```diff
@@ -29,17 +30,28 @@ const getUserRoles = (res: Response): string[] => {
 export default function routes(router: Router): Router {
   const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
 
-  get('/', (req, res) => {
+  get('/', async (req, res) => {
     res.locals.breadcrumbs.lastItem.href = undefined
 
-    const userRoles = getUserRoles(res)
+    // a prison case load would have locations, e.g. wings or house blocks
```

Comment:
This will work but its kind of a side effect in that locations are not returned from the `me` endpoint for CADM_I - I thought we would have the caseload ID in the session so could just check that.  However happy to leave as is.

Reply:
I found that `CADM_I` was the active case load ID (not a role / or maybe _also_ a role) so wasn't sure how else we'd know that the "current session" isn't a proper location…

I got the idea of using these locations from the DPS home page’s tiles. Even before me change, there had been a condition on the locations within the active case load:

https://github.com/ministryofjustice/digital-prison-services/commit/49d98598e22c50d42180818a12f85c14c280c918#diff-f8a9492c5a04c0a52f307b83958be45fd23f684f6dafeb35e41a3df8aeaaa675L78